episodes/transfer_data.md

```diff
@@ -72,7 +132,81 @@ Write up your thoughts in our shared collaborative document.
 
 :::
 
-::::{dropdown} Push GitHub code
+## Share and manage code with GitHub
+
+GitHub is a popular platform for creating, storing, managing and sharing code with collaborators. For general guides for using the Git extensions for JupyterLab and RStudio, please refer to [here](https://blog.reviewnb.com/jupyterlab-git-extension/#git-workflow) and [here](https://docs.posit.co/ide/user/ide/guide/tools/version-control.html#integrations), respectively.
+
+:::{note}
+We recommend performing git operations using the HTTPS protocol and not SSH, since on the hub HTTPS will always work behind a firewall or proxy.
+:::
+
+When a user pulls and pushes code to a remote repository on GitHub, they will be prompted to provide their GitHub credentials.
```

Comment:
There is quite a leap implied here: if the reader understands what is meant by "push", "pull", and "remote repository" here -- and understands why we would want to do this with code on a hub -- we can assume that they do not need the "what is GitHub" sentence in the previous paragraph...